src/View/ViewBuilder.php

```diff
@@ -0,0 +1,332 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         3.1.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\View;
+
+use Cake\Core\App;
+use Cake\Event\EventManager;
+use Cake\Network\Request;
+use Cake\Network\Response;
+use Cake\View\View;
+use Cake\View\Exception\MissingViewException;
+
+
+/**
+ * Provides an API for iteratively building a view up.
+ *
+ * Once you have configured the view and established all the context
+ * you can create a view instance with `build()`.
+ */
+class ViewBuilder
+{
+    /**
+     * The subdirectory to the view.
+     *
+     * @var string
+     */
+    protected $viewPath;
```

Comment:
I like _ for visibility. In case it is removed at one place, it should across the framework (consistent)?